code/game/machinery/slotmachine.dm

```diff
@@ -68,10 +68,9 @@
 			coinvalues["[cointype]"] = C.get_item_credit_value()
 			qdel(C) //Sigh
 
-/obj/machinery/computer/slot_machine/Destroy()
+/obj/machinery/computer/slot_machine/on_deconstruction(disassembled)
 	if(balance)
 		give_payout(balance)
-	return ..()
```

Comment:
why not call parent here but do so later on?

Reply:
There does not seem to be a standard here. The implementations of `on_deconstruction` sometimes call parent, sometimes they don't. It feels very random.

Reply:
It's not random per day. on_deconstruct() is an empty proc that does nothing by default so you should check and call parent only if nessassary

Reply:
Ah, I more of meant, half of the machines call the empty parent, and half of them does not.

Reply:
I will double check though, to make sure nothing is broken with slot machines not calling them.

Reply:
Double checked, the parent is empty, so me not calling it here is all good.

Reply:
aight, vibes. I do wish we were consistent about this tho